packages/core/upload/server/services/image-manipulation.js

```diff
@@ -97,10 +96,10 @@ const optimize = async (file) => {
     const transformer = sharp();
     // reduce image quality
     transformer[format]({ quality: sizeOptimization ? 80 : 100 });
-    // rotate image based on EXIF data
-    if (autoOrientation) {
```

Comment:
removing autoOrientation flag is not the solution.
Goal was to figure out why autoOrientation is set for A but not for B.
aka it should rotate everything or noting but this flag needs to stay intact.